server/src/routes/api/v1/dogParksRouter.js

```diff
@@ -44,4 +43,25 @@ dogParksRouter.get("/:id", async (req, res) => {
     }
 })
 
+dogParksRouter.post("/:id", async (req, res) => {  
+    try {
+        const { body } = req
+        const userId = req.user.id
+        body.review.userId = userId
+        const { id } = req.params
+        body.review.dogParkId = id
+        const cleanedInput = cleanUserInput(body.review)
+        const review = await Review.query().insertAndFetch(cleanedInput)
+        const park = await DogPark.query().findById(id)
+        const serializedPark = await DogParksSerializer.detailsForShow(park)
+        res.set({"Content-Type": "application/json"}).status(201).json({ park: serializedPark })
```

Comment:
the endpoint is creating a new review, so following proper HTTP protocol we should return back the newly persisted Review. Once received in react, we then add the new review to state with the rest of the reviews. React already knows the dog park and other reviews, so this is adding extra bulk in the response